index.html

```diff
@@ -0,0 +1,174 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Lato&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="./style.css">
+    <script src="./scripts.js" defer></script>
+    <title>Tennis Summit 2023</title>
+</head>
+
+<body>
+    <header class="header">
+        <nav class="navbar">
+            <div class="navbar1">
+                <ul>
+                    <li><a href="https://www.facebook.com/profile.php?id=100012707058550"><img src="./img/facebook.png"
+                                alt="Facebook social Icon"></a></li>
+                    <li><a href="https://twitter.com/dheerajarya"><img src="./img/twitter_icon.png"
+                                alt="Twitter Social Icon"></a></li>
+                    <li>English</li>
+                    <li>My Page</li>
+                    <li>Logout</li>
+
+                </ul>
+            </div>
+            <div class="navbar2">
+                <div class="logo"><a href="./index.html"><img src="./img/logo.jpg" alt="Tennis Summit Logo"></a></div>
+                <ul class="navbar2-menu">
+                    <li><a href="./about.html">About</a></li>
+                    <li>Program</li>
+                    <li>Join</li>
+                    <li>Sponsor</li>
+                    <li>News</li>
+                </ul>
+                <div class="campaign">
+                    <button class=button-campaign>CC Campaign</button>
+                </div>
+            </div>
+        </nav>
+    </header>
+
+    <div class="menu">
+        <a href="./index.html">Home</a>
+        <a href="./about.html">About</a>
+    </div>
+    <section class="section">
+        <div class="hamburger">
+            <div class="hamburger-line"></div>
+            <div class="hamburger-line"></div>
+            <div class="hamburger-line"></div>
+        </div>
+        <div class="page-container">
+            <div class="section-container">
+                <div class="section-heading">
+
+                    <h1>
+                        "Hello! Tennis Lovers"
+                    </h1>
```

Comment:
- [ ] This is not the main heading on this page. Please change the `h1` tags. According to semantic HTML, `h1` tags are given to the main heading on a page and there can only be one such tag per page. You can however have more than one h2, h3 ... per page

Reply:
Hello Francis,

Done.  Thanks